static/src/js/containers/TimelineContainer/__tests__/TimelineContainer.test.js

```diff
@@ -175,4 +176,12 @@ describe('TimelineContainer component', () => {
       }
     })
   })
+
+  test('Does not show the timeline if it is on the saved projects page', () => {
```

Comment:
@eudoroolivares2016 It's probably too much to move to RTL from Enzyme here right?

Reply:
Thought about it felt a bit out of scope. At this point I think we can do that another time there are some other outstanding timeline tickets that would be a good opportunity to get rid of that technical debt